diff --git a/askap/pipelinetasks/apps/CMakeLists.txt b/askap/pipelinetasks/apps/CMakeLists.txt
index c247dde..7f9783e 100644
--- a/askap/pipelinetasks/apps/CMakeLists.txt
+++ b/askap/pipelinetasks/apps/CMakeLists.txt
@@ -1,7 +1,5 @@
 set (_progs
-cflag
 mssplit
-cmodel
 )
 
 if (OPENSSL_FOUND)
@@ -31,18 +29,11 @@ foreach (prog ${_progs})
 endforeach(prog ${_progs})
 
 # build tDataAccessors
-add_executable(tDataAccessors tDataAccessors.cc)
 target_include_directories(pipelinetasks PUBLIC
   $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}>
   $<INSTALL_INTERFACE:include>
   $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/askap/pipelinetasks>
 # skymodel include
-  $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/_build/askap-sms/skymodel/client-cpp/smsclient/ice-generated-cpp>
-  $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/_build/askap-sms/skymodel/client-cpp/smsclient>
-  $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/_build/askap-sms/skymodel/client-cpp>
-  $<INSTALL_INTERFACE:include/SkyModel>
 )
 
-target_link_libraries(tDataAccessors  askap::pipelinetasks)
 
-install (TARGETS tDataAccessors DESTINATION bin)
